sim/core/base_stats_auto_gen.go

```diff
@@ -7,12 +7,13 @@ package core
 const ExpertisePerQuarterPercentReduction = 2.500000
 
 // Crit/Hit/Haste ratings are straight percentage values in classic
+// TODO: Update Defense/Dodge/Parry rates
 const HasteRatingPerHastePercent = 1
 const CritRatingPerCritChance = 1
 const SpellCritRatingPerCritChance = 1
 const MeleeHitRatingPerHitChance = 1
 const SpellHitRatingPerHitChance = 1
-const DefenseRatingPerDefense = 1.500000
+const DefenseRatingPerDefense = 1
 const DodgeRatingPerDodgeChance = 13.800001
 const ParryRatingPerParryChance = 13.800001
 const BlockRatingPerBlockChance = 5.000000
```

Comment:
If any of the latter three is still in use, they'd have to be adapted to 1 as well, even here.